vendor/github.com/apicurio/apicurio-operators/apicurito/pkg/apis/apicur/v1alpha1/apicurito_types.go

```diff
@@ -1,19 +1,3 @@
-/*
- * Copyright (C) 2020 Red Hat, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
 package v1alpha1
```

Comment:
Shouldn't this apicurito stuff be removed from here, given that it's imported as a Go module (at least it's listed in the go.mod)?

Reply:
It is moved in this PR. This is showing the change to the vendor folder.

Reply:
Ah yep, my mistake. :+1: